Makefile

```diff
@@ -1,4 +1,7 @@
 REPORTER = spec
+CIRCLE_TEST_REPORTS=./reports
+CIRCLE_ARTIFACTS=./artifacts
```

Comment:
these shouldn't be necessary because they're only referenced in `test-ci`, which should only be run in CI